demo/regression/main.go

```diff
@@ -69,6 +69,7 @@ func testUpgrade(orch *lib.Orchestrator) (err error) {
 	orch.CheckNewBeacon(1)
 	orch.StartNode(1)
 	orch.WaitPeriod()
+	orch.WaitPeriod()
```

Comment:
We could make a for loop that waits until the node has catched up (i.e. tries to get latest beacon and if not, sleep a while and tries again) with a timeout. But I guess it's fine to start like this..